app/assets/stylesheets/application.css.scss

```diff
@@ -36,8 +36,167 @@ h2 {
   color: #ffffff;
 }
 
+h3 {
+  color: #ffffff;
+}
+
 ul {
     list-style-type: none;
     padding: 0;
     margin: 0;
 }
+
+.nine-columns li {
+	width:calc(100% / 9);
+}
+
+.stat-labels li {
+	color: #d3d3d3;
+	display: inline;
+	float:left;
+	cursor: pointer;
+}
+
+.stat-labels li:hover {
+  transition: all .1s ease-in;
+  transform: scale(1.4);
+  color: #DDE587;
+}
+
+.stat-labels li.active {
+  transition: all .1s ease-in;
+  transform: scale(1.4);
+  color: #DDE587;
+}
+
+.character-pic {
+  height: 300px;
+  width: 300px;
+}
+
+/* drag and drop character pics */
+.layout {
+    height: 100%;
+
+    * {
+        box-sizing: border-box;
+    }
+
+    body {
+        width: 100%;
+        height: 100%;
+        padding: 20px;
+        color: #111;
+        background-image: radial-gradient(ellipse farthest-corner at center, #7abcff 0%,#4096ee 100%);
+    }
+
+    h1 {
+        font-family: Lobster, Arial;
+        text-align: center;
+    }
+
+    .btn {
+        position: absolute;
```

Comment:
Properties should be ordered background-image, border, border-radius, bottom, box-shadow, color, font-family, font-size, padding, position, right, z-index